RPIPs/rpip-draft_rewards_tree.md

```diff
@@ -0,0 +1,36 @@
+---
+rpip: #<to be assigned>
+title: Rewards Tree Specification
+description: Specifies how RPL Inflation and Smoothing Pool rewards are allocated to node operators
+author: Ramana Kumar (@xrchz), Joe Clapis (@jcrtp)
+discussions-to: https://github.com/rocket-pool/rocketpool-research/issues
+status: Draft
+type: Informational
+created: 2024-04-09
+---
+
+## Abstract
+This RPIP describes the current state of the specification of the Rocket Pool rewards system (first introduced in [Redstone](https://medium.com/rocket-pool/rocket-pool-the-merge-redstone-601d9efd6b4)) for node operators, wherein ETH and RPL rewards are allocated at regular intervals to staking nodes.
+
+## Motivation
+Rocket Pool's allocation of rewards to node operators needs a clear specification that can be independently implemented and verified so that node operators know what to expect and have a basis for recourse in case of any misallocation.
+
+The rewards tree specification is currently developed in the external [Rocket Pool Research](https://github.com/rocket-pool/rocketpool-research/tree/master/Merkle%20Rewards%20System) repository. Development there began before the RPIP process was created. However, since this is a core protocol specification, its proper home is as an RPIP where it can be easily found and referred to with a clear status as an official definition of (part of) the Rocket Pool protocol. This RPIP is an informational document describing the status quo, with the intention of moving the core protocol specification into a future RPIP.
```

Comment:
"with the intention of following up with a future pDAO-ratified RPIP (which may or may not include changes)."

Reply:
Done